### DISCUSSION TITLE
Poetry stuck when installing GCP keyring plugin

### DISCUSSION THREAD

**Discussion by snthibaud** [python-poetry/poetry#6701] [Q&A]
Hello!
I have encountered the issue described here:
https://github.com/python-poetry/poetry/issues/6668

Basically, I am unable to install the GCP keyring plugin on a Raspberry Pi 4. Any ideas?

**Comment by clintonroy** [comment]
Have you tried stracing it?

**Comment by snthibaud** [comment]
@clintonroy Apologies for the late reply! I have straced it now. Still wondering what is wrong...
